benchmark_runner/common/grafana/update_grafana_last_value_mappings.py

```diff
@@ -0,0 +1,116 @@
+
+import json
+from datetime import datetime, timedelta
+
+from benchmark_runner.main.environment_variables import environment_variables
+from benchmark_runner.common.elasticsearch.elasticsearch_operations import ElasticSearchOperations
+
+
+class UpdateGrafanaLastValueMappings:
+    """
+    This class updates grafana dashboard with last value mappings from ElasticSearch
+    """
+    MAX_VERSION_LEN = 20
```

Comment:
What's this for (please document it).

Reply:
@RobertKrawitz, add document.
Prevent error messages from being displayed instead of versions in case of a connection issue